quilt/tools/command.py

```diff
@@ -143,6 +146,16 @@ def login():
 
     _save_auth(auth)
 
+def logout():
+    """
+    Become anonymous. Useful for testing.
+    """
+    auth_file = os.path.join(BASE_DIR, AUTH_FILE_NAME)
```

Comment:
unless web and client share a refresh token? in which case it would be weird to log out of the web from the cli.